core/src/main/kotlin/org/evomaster/core/output/service/TestSuiteWriter.kt

```diff
@@ -752,4 +844,21 @@ class TestSuiteWriter {
 
 
     private fun useRestAssured() = config.problemType != EMConfig.ProblemType.RPC
+
+    /**
+     * Returns a distinct List of [HttpExternalServiceAction] from the given solution
+     */
+    private fun getWireMockServers(solution: Solution<*>): List<HttpExternalServiceAction> {
```

Comment:
the name is confusing... i mean, we could return several actions for the same `Server`. so, maybe `getWireMockServerActions`?